test/server.test.js

```diff
@@ -1109,6 +1127,8 @@ test('next.ifError', function (t) {
 
     SERVER.get('/foo/:id', function tester(req, res, next) {
         process.nextTick(function () {
+            // this object fails to become JSON because it's picking up the
+            // arguments of the call stack which includes itself
             var e = new RestError({
```

Comment:
This error picks up the arguments from the call which includes itself and thus creates a cycle. Other errors nested errors but this one doesn't.